### ISSUE TITLE
VPN Unable to Set Up on API>=30

### ISSUE THREAD

**Issue by rschulman** [yggdrasil-network/crispa-android#45]
Apparently a change in Android API 30 has made accessing the `bind` syscall a seccomp violation. Trying to run the native go binary under Android panics with this error:

``` I/GoLog: Build name: yggdrasil
    Build version: 0.3.15
    Starting up...
I/GoLog: Starting switch
    Starting router
    Startup complete
I/GoLog: Starting multicast module
I/GoLog: Started multicast module
I/TetheringManager: registerTetheringEventCallback:org.westwork.idili
I/GoLog: TCP admin socket listening on 127.0.0.1:9001
W/.westwork.idili: type=1400 audit(0.0:332): avc: denied { bind } for scontext=u:r:untrusted_app:s0:c154,c256,c512,c768 tcontext=u:r:untrusted_app:s0:c154,c256,c512,c768 tclass=netlink_route_socket permissive=0 b/155595000 app=org.westwork.idili
W/.westwork.idili: type=1400 audit(0.0:333): avc: denied { read } for name="somaxconn" dev="proc" ino=306658 scontext=u:r:untrusted_app:s0:c154,c256,c512,c768 tcontext=u:object_r:proc_net:s0 tclass=file permissive=0 app=org.westwork.idili
E/Go: panic: route ip+net: netlinkrib: permission denied
E/Go: goroutine 39 [running]:
    github.com/yggdrasil-network/yggdrasil-go/src/multicast.(*Multicast).getAllowedInterfaces(0x954a8644, 0x95602460)
    	/home/ross/go/src/github.com/yggdrasil-network/yggdrasil-go/src/multicast/multicast.go:194 +0x2c0
    github.com/yggdrasil-network/yggdrasil-go/src/multicast.(*Multicast)._updateInterfaces(0x954a8644)
    	/home/ross/go/src/github.com/yggdrasil-network/yggdrasil-go/src/multicast/multicast.go:160 +0x3c
    github.com/yggdrasil-network/yggdrasil-go/src/multicast.(*Multicast)._announce(0x954a8644)
    	/home/ross/go/src/github.com/yggdrasil-network/yggdrasil-go/src/multicast/multicast.go:229 +0x42
    github.com/Arceliar/phony.(*Inbox).run(0x954a8644)
    	/tmp/gomobile-work-019624322/pkg/mod/github.com/!arceliar/phony@v0.0.0-20191006174943-d0c68492aca0/actor.go:104 +0x87
    created by github.com/Arceliar/phony.(*Inbox).restart
    	/tmp/gomobile-work-019624322/pkg/mod/github.com/!arceliar/phony@v0.0.0-20191006174943-d0c68492aca0/actor.go:130 +0x39
A/libc: Fatal signal 6 (SIGABRT), code -6 (SI_TKILL) in tid 12940 (.westwork.idili), pid 12905 (.westwork.idili)
I/Yggdrasil: Yggdrasil service started
```

**Comment by vikulin** [comment]
Duplicates https://github.com/vikulin/yggdrasil-android/issues/5

**Comment by vikulin** [comment]
fixed in v 1.7
